elements/lisk-chain/src/constants.ts

```diff
@@ -12,7 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-export const CHAIN_STATE_KEY_BURNT_FEE = 'CHAIN_STATE_KEY_BURNT_FEE';
+export const CHAIN_STATE_BURNT_FEE = 'Chain:burntFee';
```

Comment:
```suggestion
export const CHAIN_STATE_BURNT_FEE = 'chain:burntFee';
```
same for other places